test/tests/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+yum install nodejs npm
+npm -g install phantomjs
+
+yum install yum-plugin-copr
+yum copr enable lmr/Autotest
+yum install avocado
+yum install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+setenforce 0
```

Comment:
Thanks, your mentioned virt-deploy seems very good (creating vms based on teplates).

I'm recompiling to /opt because I need more testing instances of cockpit for prototype. 
Then this test will not disable selinux and cockpit will be installed via rpms, or will be compiled&installed to /usr.